colossalai/inference/modeling/models/nopadding_llama.py

```diff
@@ -439,7 +446,15 @@ def __init__(
             self.helper_layout = (
                 attn_qproj_w.dist_layout
             )  # NOTE this is a hack for the right load/shard of qkv_weight(used in _load_from_state_dict)
+            self.qkv_dict = {
+                "q_proj.weight": None,
+                "k_proj.weight": None,
+                "v_proj.weight": None,
```

Comment:
Same as above. If the structure is only used in a single-time-calling function, we don't need to set it as object attributes

Reply:
Same as above.